src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -636,10 +635,5 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures {
         _ <- dataAccess.updateExecutionBackendInfo(workflowId, call, null)
       } yield ()).failed.futureValue should be(an[IllegalArgumentException])
     }
-
-    it should "shutdown the database" in {
-      assume(canConnect || testRequired)
-      dataAccess.shutdown().futureValue
```

Comment:
Trace back this `dataAccess` variable (command click in IntelliJ). It **should** be shutdown, as SDAS is creating and testing the different connection pools.